lib/private/TemplateLayout.php

```diff
@@ -134,7 +135,14 @@ public function __construct( $renderAs, $appId = '' ) {
 			parent::__construct('core', 'layout.public');
 			$this->assign( 'appid', $appId );
 			$this->assign('bodyid', 'body-public');
-			$this->assign('showSimpleSignUpLink', $this->config->getSystemValue('simpleSignUpLink.shown', true) !== false);
+
+			/** @var IRegistry $subscription */
+			$subscription = \OC::$server->query(IRegistry::class);
+			$showSimpleSignup = $this->config->getSystemValue('simpleSignUpLink.shown', true) !== false;
```

Comment:
```suggestion
			$showSimpleSignup = $this->config->getSystemValueBool('simpleSignUpLink.shown', true) !== false;
```

If you are not going to backport it to 15.

Reply:
Fair enough